build/assets/tuned/openshift-node-performance

```diff
@@ -35,6 +35,9 @@ transparent_hugepages=never                   #  network-latency
 [irqbalance]
 # Override the value set by cpu-partitioning with an empty one
 banned_cpus=""
```

Comment:
Now looking at this, is our CRI-O logic configuring irqbalance properly when needed?

Reply:
I hop so... What do you mean exactly?

Reply:
@MarSik  When the irqs are not globally disabled we don't ban any CPUs; the pod annotation itself will hint crio to set the default_smp_affinity to all the cpus but this pod's cpu (tested)

Reply:
But isn't irqbalance a dynamic service that needs to know it should not rebalance (at runtime) to certain cores?

Reply:
cri-o will call irqbalance cli specifying the cores suitable for irqs. This is set is checked against the default_smp_affinity mask.
Before this patch the default_smp_affinity included only the reserved the CPUs, so the call to irqbalance did not do much...